scripts/android/helper.js

```diff
@@ -47,6 +47,13 @@ function addRepos(buildGradle) {
   // modify the actual line
   buildGradle = buildGradle.replace(/^(\s*)jcenter\(\)/m, modifiedLine);
 
+  var googlesMavenRepo = whitespace + 'google() // Google\'s Maven repository from cordova-plugin-firebase';
+  // Only add Google's Maven repo if it's not already there
+  var googleRepoAlreadyExists = buildGradle.test(/(google\(\))|(https:\/\/maven\.google\.com)/m);
+  if (googleRepoAlreadyExists) {
```

Comment:
Instead of setting the variable twice, along with running regex and string replacement, why don't we wrap the change to only run the code if !googleRepoAlreadyExists?  This will reduce code complexity and improve performance.